event-handlers/src/main/java/no/sikt/nva/nvi/events/evaluator/EvaluatorService.java

```diff
@@ -82,12 +83,15 @@ public Optional<CandidateEvaluatedMessage> evaluateCandidacy(URI publicationBuck
             logger.info(NON_NVI_CANDIDATE_MESSAGE, publicationId);
             return createNonNviMessage(publicationId);
         }
-        var verifiedCreatorsWithNviInstitutions =
-            creatorVerificationUtil.getVerifiedCreatorsWithNviInstitutionsIfExists(
+        var verifiedCreatorsWithNviInstitutions = creatorVerificationUtil.getVerifiedCreatorsWithNviInstitutions(
                 publication);
-        if (!verifiedCreatorsWithNviInstitutions.isEmpty()) {
+
+        var unverifiedCreatorsWithNviInstitutions = creatorVerificationUtil.getUnverifiedCreatorsWithNviInstitutions(
+            publication);
+        if (!verifiedCreatorsWithNviInstitutions.isEmpty() || !unverifiedCreatorsWithNviInstitutions.isEmpty()) {
```

Comment:
Extract to method `hasCreatorsWithNviAffiliations(JsonNode publication)`?

Reply:
We use the lists further down, but I've extracted the logic check to a method at least.